Makefile

```diff
@@ -16,30 +16,53 @@
 #  License along with this library; if not, write to the Free Software
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 #  MA 02110-1301 USA
-#
 
+ifeq ($(OS),Windows_NT)
+    detected_OS := Windows
+    SHARED_LIB_EXT := dll
+    EXTRA_OBJS := wcwidth.o
+else
+    detected_OS := $(shell uname)
+    SHARED_LIB_EXT := so
+    EXTRA_OBJS :=
+endif
 include Makefile.cfg
 
 export CC = gcc -pthread
 export CFLAGS += -I. -Wall -Os -ggdb -D_GNU_SOURCE -fPIC
-export LDLIBS += -lncursesw
-
-SONAME  := libstfl.so.0
+export LDLIBS += $(shell pkg-config --libs ncursesw iconv)
+export CFLAGS += $(shell pkg-config --cflags ncursesw iconv)
```

Comment:
Here on Debian stable, this breaks the build because `iconv` can't be found. As far as I know, GNU libc contains an implementation of iconv functions, so there's no separate library for it. The GNU iconv is a separate implementation which can be used on platforms that don't have their own. Perhaps it should only be included on Windows?

Reply:
https://github.com/newsboat/stfl/pull/7/commits/034fb2a5f102c37ff928fbb54dcc797989e162a4 fixed it, thanks!